playground/ssr-noexternal/vite.config.js

```diff
@@ -15,5 +15,6 @@ export default defineConfig({
     rollupOptions: {
       external: ['@vitejs/test-external-cjs'],
     },
+    ssr: 'src/entry-server.js', // for 'all'
```

Comment:
I removed the changes to ssr-noexternal, this was a testing change that was leaked. I still don't know what is the best way to test `build --app` (`--all` is outdated here) for all our tests. We shouldn't be testing everything twice just because of this. Maybe we can move them all to `build --app` once we move Environment API out of experimental.